main.js

```diff
@@ -75,7 +75,7 @@ if (handleStartupEvent()) {
   console.log('OpenBazaar started on Windows...');
 }
 
-const serverPath = `${__dirname}${path.sep}..${path.sep}OpenBazaar-Server${path.sep}`;
+const serverPath = `${__dirname}${path.sep}..${path.sep}openbazaar-go${path.sep}`;
```

Comment:
Hmm... now that I think about it, an alternative would be to have the bundled app start the client with a command line flag (e.g. `app-type="packaged"`).

That would give the added benefit in that we could also pass in a flag when it's a stand-alone client *exectuable*  (e.g. `app-type="packaged-client-only"`) since I remember there being a use-case for the client knowing it's being run that way (as opposed to from source).

What do you think of that? (for next release)